src/controllers/ordersController.js

```diff
@@ -0,0 +1,29 @@
+const ordersService = require("../services/ordersService");
+
+const addUserAddress = async (req, res) => {
+  try {
+    const userId = req.userId;
+    const { address, postalCode } = req.body;
+
+
+    if (!userId || !address || !postalCode) {
+      return res.status(400).json({ message: 'USER_INPUT_DATA_IS_NOT_ENOUGH' });
+    }
+
+    await ordersService.addUserAddress(userId, address, postalCode);
+
+    return res.status(202).json({
```

Comment:
202를 사용한 이유를 template에 남겨주세요.
```suggestion
    return res.status(200).json({
```

Reply:
200으로 변경드렸습니다!